virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -163,6 +163,8 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 
 //CNV-28944 Release note: Preview Cluster level eviction strategy change
 
+//CNV-29940 Release note: Preview UI Bridged network interface hot-plug for VMs
+* You can hot-plug a bridge network interface and then restart or live migrate the virtual machine (VM) to apply the change. Hot unplugging is supported only for VMs created with {VirtProductName} 4.13 or later. 
```

Comment:
Should "hot-plug" be "hot plug"? I was observing other instances in the repo where "hot plug" is used as a verb. For eg: https://docs.openshift.com/container-platform/4.13/virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.html#virt-about-hot-plugging-virtual-disks_virt-hot-plugging-virtual-disks
> When you hot plug a virtual disk, you attach......
> After you hot plug a virtual disk, it remains.......